### ISSUE TITLE
handcuff pulling

### ISSUE THREAD

**Issue by Iamgoofball** [NTStation/NTstation13#192]
_Issue by **[KazeEspada](https://github.com/KazeEspada)** from Thursday Nov 21, 2013 at 23:44 GMT_
_Originally opened as https://github.com/tgstation/-tg-station/issues/1840_

---
##### Revision that this bug was encountered on (REQUIRED!! Type 'show-server-revision' ingame to find out - note that 'current revision' isn't helpful!):

2013-11-15
16b2bc833fd647e0abc347d7661111096d2d48ed
##### Which server were you playing on? (REQUIRED!! e.g. TG-Sibyl1, TG-Sibyl2, Badgerstation, Facepunch, Pinkstation, /VG/... etc.)

Basil
###### What did you expect to happen?

To get removed from the on top of the trashcan
###### What happened instead?

My captors could not pull or grab me off the disposals bin
###### Why is this bad/What are the consequences?

They had to kill me to move me.
##### Steps to reproduce the problem (remember to be specific! Those little details can save us hours of work!):

1.get cuffed(root cause)
2.climb into disposals(probably should be checking for cuffs) or something else?
3.get ejected by angry sec officers
4.they can't pull you off
5.honk
##### Possibly related stuff (which gamemode was it? What were you doing at the time? Was anything else out of the ordinary happening?):

traitor
##### Anything else?

nope

**Comment by Iamgoofball** [comment]
_Comment by **[KazeEspada](https://github.com/KazeEspada)** from Friday Nov 22, 2013 at 00:46 GMT_

---

higgs bugson, it did not happen in the round after this one.

**Comment by Iamgoofball** [comment]
_Comment by **[Aranclanos](https://github.com/Aranclanos)** from Friday Nov 22, 2013 at 02:36 GMT_

---

> Problem Description

please answer that one. With more than 10 words.

**Comment by Iamgoofball** [comment]
_Comment by **[KazeEspada](https://github.com/KazeEspada)** from Friday Nov 22, 2013 at 03:05 GMT_

---

I stood on a disposals bin and no one could pull or grab me off.

**Comment by Iamgoofball** [comment]
_Comment by **[Aranclanos](https://github.com/Aranclanos)** from Friday Nov 22, 2013 at 03:27 GMT_

---

they were able to pull/grab you before that?
Did you have an encounter with rolling chairs or maybe space vines before this happened?

**Comment by Iamgoofball** [comment]
_Comment by **[KazeEspada](https://github.com/KazeEspada)** from Friday Nov 22, 2013 at 04:11 GMT_

---

no chairs or space vines involved, and they were dragging me around until someone broke their grip and i climbed into disposals and then they ejected me and could not remove me from the top of the disposals bin.

**Comment by Iamgoofball** [comment]
_Comment by **[MrPerson](https://github.com/MrPerson)** from Saturday Nov 23, 2013 at 01:02 GMT_

---

Were you perhaps grabbed or have something else that would cause you to be unable to move beyond handcuffs+dragging? Did somebody else drag you off maybe? Can you come up with a way to reproduce this issue? It's going to be hard to solve with just this.

**Comment by Iamgoofball** [comment]
_Comment by **[KazeEspada](https://github.com/KazeEspada)** from Tuesday Dec 17, 2013 at 00:05 GMT_

---

Saw this again. No trashcan this time, but it was a handcuffed ling.

**Comment by Iamgoofball** [comment]
_Comment by **[Malkevin](https://github.com/Malkevin)** from Friday Jan 10, 2014 at 19:44 GMT_

---

Possibly related.

I got grabbed by space weed, a borg welded me out of the vines, I was able to move and stuff fine.
Jumped out an air lock but only travelled one tile away from the airlock, threw an object which only moved me one more tile instead of the usual moving me along continuously.

A borg that jumped out the airlock after me moved as normal

**Comment by Iamgoofball** [comment]
_Comment by **[GarrusVakarian](https://github.com/GarrusVakarian)** from Sunday Jan 26, 2014 at 16:14 GMT_

---

Just encountered the problem on Sibyl.

Server Revision: 
2014-01-24
8660698ce0133d4ed8e2a065cb4fb83885e6af52

Handcuffed engineer ran into engineering. I flashed him. I tried pulling him via both ctrl-click and right click > pull. Neither worked. Kept flashing him. Had someone else try to pull him. He couldn't either. Ended up having to kill him.

Not sure what happened before the engineer came to engineering.

Forgot to mention: other guy bucklecuffed him by building a chair underneath him. Could move him fine after. Unsure if I could move him because he was dead or because he was bucklecuffed
